src/lib/core/authorization/AuthorizationManager.ts

```diff
@@ -321,9 +324,17 @@ export class AuthorizationManager {
    * @param response - The error response from a Globus service.
    * @param execute - Whether to execute the handler immediately or return a function that can be executed later.
    */
-  handleErrorResponse(response: Record<string, unknown>, execute?: true): void;
-  handleErrorResponse(response: Record<string, unknown>, execute?: false): () => void;
-  handleErrorResponse(response: Record<string, unknown>, execute = true) {
+  handleErrorResponse(
+    response: Record<string, unknown>,
+    execute?: true,
+    overrides?: Overrides,
+  ): void;
+  handleErrorResponse(
+    response: Record<string, unknown>,
+    execute?: false,
+    overrides?: Overrides,
+  ): () => void;
+  handleErrorResponse(response: Record<string, unknown>, execute = true, overrides?: Overrides) {
```

Comment:
I'd like to avoid this signature expanding, especially since  the `execute` property was meant to be a rarely used parameter, but I thinking passing parameters here would be much more common.

My initial thought is this should just swap to an `options` parameter, `handleErrorResponse(response, options: {
execute?: boolean, additionalParams: ... } | boolean);` (the `| boolean` making sure the signature stays backwards compatible for this version).


But going to think about this a bit... I've actually found in a couple of use cases where getting the handler back as `() => void | null` from a method would be preferred over calling `handleErrorResponse(res, false)` to give callers a better idea if a useful handler was actually found based on the response.